cluster-autoscaler/README.md

```diff
@@ -92,12 +92,12 @@ target ETA and the actual releases.
 
 | Date       | Maintainer Preparing Release | Backup Maintainer | Type  |
 |------------|------------------------------|-------------------|-------|
-| 2024-07-18 | x13n                         | MaciekPytel       | patch |
-| 2024-08-21 | MaciekPytel                  | gjtempleton       | 1.31  |
-| 2024-09-18 | gjtempleton                  | towca             | patch |
-| 2024-11-20 | towca                        | BigDarkClown      | patch |
-| 2024-12-18 | BigDarkClown                 | x13n              | 1.32  |
-| 2025-01-22 | x13n                         | MaciekPytel       | patch |
+| 2025-06-11 | gjtempleton                  | jackfrancis       | 1.33  |
+| 2025-07-16 | jackfrancis                  | towca             | patch |
```

Comment:
Any chance I could take the 6/11 slot? I'll be on holiday on 7/16?

Reply:
Works for me, updated. If there are more vacation conflicts, we can also move dates a bit - I was aiming for 1/month, but it should be fine to do them a week sooner/later.

Reply:
thank you!